app-frontend/src/assets/styles/sass/components/_searchselect.scss

```diff
@@ -32,21 +32,23 @@
 
 .searchselect-clear {
   border-radius: 50%;
-  color: $gray;
-  border: 1px solid $gray-lighter;
-  background: $gray-lighter;
+  color: $white;
```

Comment:
Not sure where exactly this is happening css wise but the datasource select menu is covering the input which make using the filtering option a bit difficult.

![image](https://user-images.githubusercontent.com/2442245/48154798-81c65d00-e297-11e8-8e8c-7a27a121ebb0.png)